protocol/contracts/voting/ElderCandidateClaimFlow.sol

```diff
@@ -0,0 +1,96 @@
+//SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+
+/**
+ * @title Track Elder candidates and endorsements.
+ * 
+ * @notice Capture votes for temple elders on chain, allows those who are
+ * 
+ *   - Add their nomination
+ *   - Endorse candidates
+ */
+contract ElderCandidate is ERC721, Ownable {
+    using Strings for uint256;
+    using Counters for Counters.Counter;
+
+    /// @notice endorsements per nomination
+    mapping(address => mapping (uint256 => bool)) public endorsementsBy;
+
+    /// @notice metadata per nomination
+    mapping(uint256 => Candidate) public candidates;
+
+    /// @dev merkle tree root of (address, claimShare)
+    bytes32 public merkleRoot;
+
+    struct Candidate {
+        string discordHandle;
+        string twitterHandle;
+        uint256 endorsements;
+    }
+
+    constructor() ERC721("Temple Elder Candidate", "ELDER CANDIDATE") {
+    }
+
+    function claimCandidate(
```

Comment:
the merkle proof is to verify that someone is a current elder and no longer just a candidate?

Reply:
The idea with this approach is everyone in discord _is_ a candidate already. They just have to claim it.

Stops people scamming/spamming people up as candidates.